re2/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.0)
 project(re2 CXX C)
 
+add_definitions(-Wno-narrowing)
```

Comment:
I think there is a separate fix for re2, which I thought we'd merged, maybe not. Also, doesn't this only silence the warning on older gcc, newer will still error?

I don't think we should take this change, at least not right now.